index.html

```diff
@@ -57,6 +72,11 @@ <h4>LET’S CONNECT</h4>
       </header>
     </section>
 
+
+    <div id="popup2" class="popup2">
+      <!-- render porducts here -->
+    </div>
```

Comment:
Good job identifying the point in your code that should be dynamically generated. 👍🏿 However the actual implementation was not properly done. 

As it is required for you to generate the work sections dynamically with JavaScript, you are also required to remove the section from your HTML file. 🗑️ 

- [ ] Please remove the `Works section` from your HTML file and make sure it is dynamically generarted with JavaScript.✅